src/package-lock.json

```diff
@@ -0,0 +1,5865 @@
+{
```

Comment:
There shouldn't be a package-lock in `/src`, it would have been generated when running `npm install` from `src`, it should be run where a matching `package.json` file is.

Reply:
removed package-lock.json from /src in latest commit